training/instructlab/Makefile

```diff
@@ -31,7 +31,8 @@ nvidia: instructlab
 .PHONY: amd
 amd: instructlab
 	rm -rf ../build/instructlab-$@
-	"${CONTAINER_TOOL}"  build --squash-all -t oci:../build/instructlab-$@ -f instructlab/containers/rocm/Containerfile instructlab
+	"${CONTAINER_TOOL}"  build --squash-all -f instructlab/containers/rocm/Containerfile instructlab
+	# "${CONTAINER_TOOL}"  build --squash-all -t oci:../build/instructlab-$@ -f instructlab/containers/rocm/Containerfile instructlab
```

Comment:
should we do a -t quay.io/ai-labs/instructlab-amd:latest here?

Reply:
I think we need to revert this or hack something here due to a failure. I think i have a gist of how things are working now